polyssifier/polyssifier.py

```diff
@@ -289,8 +289,45 @@ def fit_clf(args, clf_name, val, n_fold, project_name, save, scoring):
             clf)  # fitted clf
 
 
+def create_multivariate(data, degree):
+    '''
+    :param data: the data (numpy matrix) which will have its data vectors raised to powers
+    :param degree: the degree of the polynomial we wish to predict
+    :return: a new data matrix of the specified degree (for polynomial fitting purposes)
+    '''
+
+    #First we make an empty matrix which is the size of what we wish to pass through to linear regress
+    height_of_pass_through = data.shape[0]
+    width_of_pass_through = degree * data.shape[1]
+    to_pass_through = np.zeros(shape=(height_of_pass_through, width_of_pass_through))
+
+    #These are the width and height of each "exponeneted" matrix
+    height_exponential_matrix = data.shape[0]
+    width_exponential_matrix = data.shape[1]
+
+    for i in range(degree):
+        to_add_in = exponent_matrix(data, (i + 1))
+        for j in range(height_exponential_matrix):
+            for k in range(width_exponential_matrix):
+                to_pass_through.itemset((j, k + i * width_exponential_matrix), (to_add_in.item(j, k)))
+    return to_pass_through
+
+def exponent_matrix(matrix, exponent):
```

Comment:
This function is not needed, you can just make the matrix a numpy array and do
`matrix ** exponent`